### PR TITLE
Re-emit events for default backend - 404 errors

### PR DESCRIPTION
**Pull request by monsha**

### CODE REVIEW

**Review by OpakAlex — APPROVED** [review]

File: lib/tasks/chattermill_reemit_events.rake [hunk 1]
```diff
@@ -18,7 +18,7 @@ namespace :chattermill do
 
     puts "Searching failed events between #{from} and #{to}"
 
-    statuses = ['%502 Bad Gateway%', '%504 Gateway Time-out%', '%something went wrong (500)%']
+    statuses = ['%502 Bad Gateway%', '%504 Gateway Time-out%', '%something went wrong (500)%', '%default backend - 404%']
```
Comment (duboff) [inline lib/tasks/chattermill_reemit_events.rake:21]: can we just say `404`? Not sure the `default backend` option will be there

Comment (monsha) [reply lib/tasks/chattermill_reemit_events.rake:21]: It's no safe, because It would looks for any '404' in the payload.

Comment (duboff) [reply lib/tasks/chattermill_reemit_events.rake:21]: OK, fair enough

**Review by duboff — APPROVED** [review]
👍

**Pull request merged**